x-pack/plugins/enterprise_search/public/applications/app_search/components/library/library.tsx

```diff
@@ -228,6 +231,28 @@ export const Library: React.FC = () => {
           <Result {...props} actions={actions} shouldLinkToDetailPage />
           <EuiSpacer />
 
+          <EuiSpacer />
+          <EuiTitle size="s">
+            <h3>With a drag handle</h3>
+          </EuiTitle>
+          <EuiSpacer />
+          <EuiDragDropContext onDragEnd={() => {}}>
+            <EuiDroppable spacing="m" droppableId="DraggableResultsTest">
+              {[1, 2, 3].map((_, i) => (
+                <EuiDraggable
+                  spacing="m"
+                  key={`draggable-${i}`}
+                  index={i}
+                  draggableId={`draggable-${i}`}
+                  customDragHandle
+                >
+                  {(provided) => <Result {...props} dragHandleProps={provided.dragHandleProps} />}
```

Comment:
From what I understand from the docs, `customDragHandle` + render prop is what gives us the magic for a custom drag handle (vs the entire component being draggable):

https://github.com/elastic/kibana/pull/94652

Reply:
Nit:

```jsx
{({dragHandleProps}) => <Result {...props} dragHandleProps={dragHandleProps} />}
```